app/controllers/concerns/signature_verification.rb

```diff
@@ -109,11 +118,15 @@ def verify_signature(account, signature, compare_signed_string)
   end
 
   def build_signed_string(signed_headers)
-    signed_headers = 'date' if signed_headers.blank?
+    signed_headers = (signature_algorithm == 'hs2019' ? '(created)' : 'date') if signed_headers.blank?
 
     signed_headers.downcase.split(' ').map do |signed_header|
       if signed_header == Request::REQUEST_TARGET
         "#{Request::REQUEST_TARGET}: #{request.method.downcase} #{request.path}"
+      elsif signature_algorithm == 'hs2019' && signed_header == '(created)'
```

Comment:
spec says to throw an error if the remote server attempts to use `(created)` with the rsa-sha256 algorithm.